Doc/library/curses.rst

```diff
@@ -786,7 +787,7 @@ the following methods and attributes:
             window.chgat(y, x, num, attr)
 
    Set the attributes of *num* characters at the current cursor position, or at
-   position ``(y, x)`` if supplied. If no value of *num* is given or *num* = ``-1``,
+   position ``(y, x)`` if supplied. If no value of *num* is given or ``-1`` is given,
```

Comment:
I don't particularly remember this, but I suspect my logic was that we don't usually write "arg = <something>" when describing argument values, since that looks like code and thus is potentially confusing in this context.  I like the "if num is not given or is -1" phrasing.  Oddly enough, I think it might read better in the form "if num is -1 or is not given", but either one is fine with me.